client/reader/stream/reader-sidebar-followed-sites/index.jsx

```diff
@@ -0,0 +1,96 @@
+import { localize } from 'i18n-calypso';
+import { map } from 'lodash';
+import PropTypes from 'prop-types';
+import { Component } from 'react';
+import { connect } from 'react-redux';
+import { isEligibleForUnseen } from 'calypso/reader/get-helpers';
+import { isFollowingOpen } from 'calypso/state/reader-ui/sidebar/selectors';
+import { recordReaderTracksEvent } from 'calypso/state/reader/analytics/actions';
+import { hasReaderFollowOrganization } from 'calypso/state/reader/follows/selectors';
+import getReaderFollowedSites from 'calypso/state/reader/follows/selectors/get-reader-followed-sites';
+import isFeedWPForTeams from 'calypso/state/selectors/is-feed-wpforteams';
+import isSiteWPForTeams from 'calypso/state/selectors/is-site-wpforteams';
+import ReaderSidebarFollowingItem from './item';
+import '../style.scss';
+
+export class ReaderSidebarFollowedSites extends Component {
+	constructor( props ) {
+		super( props );
+		this.state = {
+			sitePage: 1,
+		};
+	}
```

Comment:
Could be simplified as well:

```suggestion
	state = {
		sitePage: 1,
	};
```